packages/workspace/src/browser/workspace-input-dialog.ts

```diff
@@ -0,0 +1,49 @@
+/********************************************************************************
+ * Copyright (C) 2019 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { inject, injectable } from 'inversify';
+import { SingleTextInputDialog, SingleTextInputDialogProps } from '@theia/core/lib/browser';
+
+@injectable()
+export class WorkspaceInputDialogProps extends SingleTextInputDialogProps {
+    /**
+     * The general `path` to be displayed in the dialog.
+     */
+    path: string;
```

Comment:
Why do not we use `path: Path` (where `Path` is from `@theia/core/core/lib/common/path`)? No need to clarify the property with the _"The general `path`"_ JSDoc which is not clear by the way.